### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#584] [General]
# index

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/

**Comment by chai2010** [comment]
总算有了留言功能～

**Comment by arreychen** [comment]
支持一下

**Reply by arreychen** [reply]
开始学起来

**Comment by yaomogui** [comment]
太棒了!

**Comment by Engrtao** [comment]
已经支持

**Comment by codingknees** [comment]
mark

**Comment by ZzzJing** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Reply by Henry-jie** [reply]
快点回复

**Comment by lucas007007** [comment]
感谢开源。这本书给我巨大的帮助。

**Comment by hu-keyu** [comment]
学起来

**Comment by ZzzJing** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by changkaiyan** [comment]
您好，我将尽快回复您的邮件。如果您的邮件无需回复，则收到本回复表明我已收到您的邮件。请勿回复，谢谢合作。This is an automatic reply, confirming that your e-mail was received.Thank you.
